src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/readthreading/ReadThreadingAssembler.java

```diff
@@ -177,6 +183,14 @@ public AssemblyResultSet runLocalAssembly(final AssemblyRegion assemblyRegion,
         // print the graphs if the appropriate debug option has been turned on
         if ( graphOutputPath != null ) { printGraphs(nonRefSeqGraphs); }
         if ( graphHaplotypeHistogramPath != null ) { haplotypeHistogram.add((double)resultSet.getHaplotypeCount()); }
+        if ( debugActiveRegionOutputStream != null ) {
+            debugActiveRegionOutputStream.getNewLineBuilder()
+                .setRow(new String[]{
+                        assemblyRegion.getSpan().toString(),
```

Comment:
To be clear, I believe this outputs the active region determined by `isActive()`, prior to trimming down to the span of assembled variants, and not including the padding for assembly.  Is that as intended?

Reply:
It is indeed intended. As the extended span is useful but often confusing as it will leave you guessing about why variants at the edges were dropped. This is intended to be a representation of what variants actually get meaningfully assembled at each of these spots.

Reply:
It was intentional, I have added a second column for the assembly window as well.